build-support/emp3-gradle-plugin/src/main/resources/common.gradle

```diff
@@ -14,7 +14,11 @@ ext {
     versionFile = new File(project.rootDir, 'gradle.properties')
 
     version_nasa_worldwind_sdk = "0.8.0-SNAPSHOT"
-    version_cmapi              = "${version}"
+    //CMAPI values
+    version_cmapi              = "1.1.0.0"
+    cmapi_group_name           = "cmapi-kotlin"
+    cmapi_jvm_artifact_name    = "cmapi-multi-jvm"
```

Comment:
`cmapi-multi-jvm` is a leftover from our previous name, and I think for clarify the jvm artifact name _should_ be `cmapi-kotlin-jvm`.  If this is not an easy fix, please create a ticket to address in the future.